server/src/main/java/org/elasticsearch/search/vectors/KnnVectorQueryBuilder.java

```diff
@@ -134,42 +141,66 @@ public KnnVectorQueryBuilder(
         float[] queryVector,
         Integer k,
         Integer numCands,
+        Float visitPercentage,
         RescoreVectorBuilder rescoreVectorBuilder,
         Float vectorSimilarity
     ) {
-        this(fieldName, VectorData.fromFloats(queryVector), null, null, k, numCands, rescoreVectorBuilder, vectorSimilarity);
+        this(
+            fieldName,
+            VectorData.fromFloats(queryVector),
+            null,
+            null,
+            k,
+            numCands,
+            visitPercentage,
+            rescoreVectorBuilder,
+            vectorSimilarity
+        );
     }
 
     public KnnVectorQueryBuilder(
         String fieldName,
         QueryVectorBuilder queryVectorBuilder,
         Integer k,
         Integer numCands,
+        Float visitPercentage,
```

Comment:
We're using `Float` here - I think that's the right choice for an optional value.